src/main/java/com/useful/ucars/PlaceManager.java

```diff
@@ -29,5 +31,30 @@ public static Boolean placeableOn(String materialName, byte data) {
 		}
 		return placeable;
 	}
-
+	
+	public static Boolean placeableOn(Block block) {
+		Boolean placeable = false;
+		if (!ucars.config.getBoolean("general.cars.roadBlocks.enable")) {
+			return true;
+		}
+		List<String> rBlocks = ucars.config
+				.getStringList("general.cars.roadBlocks.ids");
+		for (String raw : rBlocks) {
+			final String[] parts = raw.split(":");
+			if (parts.length < 1) {
+			} else if (parts.length < 2) { //New configs and blocknames
+				if (ItemStackFromId.equals(raw,block.getType().name().toUpperCase(),block.getData()) || block.getType().name().toUpperCase().contains(raw)) {
```

Comment:
I'm not sure the contains(...) is the right check here. For example if you specify COBBLESTONE to be a road block you will also be inadvertently specifying COBBLESTONE_WALL and COBBLESTONE_STAIRS which you may not want

Reply:
Why also can't this call to the "isBlockEqualToConfigIds" method to avoid code duplication?

Reply:
Mhh... To keep the required config changes as minimal as possible one could require something like "!WOOL" to add all wool varieties (or cobblestone varieties etc). Or do you have a better idea?

Reply:
I think just leave it so that they have to specify all the ones the want? I don't see the problem with that